src/Tabs.tsx

```diff
@@ -1 +1,40 @@
-// implement a component here
+import React from 'react';
+import './Tabs.scss';
+
+type Props = {
+  tabsList : Tab[];
+  selectedTabId : string;
+  onTabSelected : (tab : Tab) => void,
+};
+
+export const Tabs : React.FC<Props> = ({
+  tabsList,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  return (
+    <div className="container">
+      {tabsList.map((singleTab) => (
+        <>
+          <div
+            key={singleTab.id}
+            className="tab"
+          >
+            <button
+              type="button"
+              className="tab__pick"
+              onClick={() => {
+                onTabSelected(singleTab);
```

Comment:
Will be better to call this function only if singleTab id is not equal to selected tab id